test-case-reporting/.eslintrc.js

```diff
@@ -0,0 +1,99 @@
+/**
```

Comment:
Instead of duplicating the existing configuration, we can just extend it here and add only the `project` option, like so:

```javascript
/**
 * Copyright 2020 The AMP HTML Authors. All Rights Reserved.
 *
 * Licensed under the Apache License, Version 2.0 (the 'License');
 * you may not use this file except in compliance with the License.
 * You may obtain a copy of the License at
 *
 *      http://www.apache.org/licenses/LICENSE-2.0
 *
 * Unless required by applicable law or agreed to in writing, software
 * distributed under the License is distributed on an 'AS-IS' BASIS,
 * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 * See the License for the specific language governing permissions and
 * limitations under the License.
 */

const path = require('path');

module.exports = {
  extends: ['../.eslintrc-ts.js'],
  "parserOptions": {
    "project": path.resolve(__dirname, './tsconfig.json')
  },
};
```

Reply:
Done.